ref/general/metrics-catalog.adoc

```diff
@@ -11,16 +11,18 @@
 :seo-description: The metrics contained in this reference list are all available for Open Liberty. Use metric data to effectively monitor the status of your microservice systems.
 :page-layout: general-reference
 :page-type: general
-:base-metric-features: feature:mpMetrics-2.2[]
-:vendor-metric-features: feature:mpMetrics-2.2[] and feature:monitor-1.0[]
-:ft-metric-features: feature:mpMetrics-2.2[], feature:monitor-1.0[], and feature:mpFaultTolerance-2.0[]
+:base-metric-features: feature:mpMetrics-2.3[]
+:vendor-metric-features: feature:mpMetrics-2.3[]
+:ft-metric-features: feature:mpMetrics-2.3[] and feature:mpFaultTolerance-2.0[]
 = Metrics reference list
 
 The metrics reference list includes all the metrics that are available for Open Liberty. Use metric data to effectively monitor the status of your microservice systems.
 
 Metrics are emitted from a number of different places. You can obtain them from applications, the Open Liberty runtime, and the Java virtual machine (JVM). They can be gathered and stored in database tools, such as link:https://prometheus.io/[Prometheus], and displayed on dashboards, such as link:https://grafana.com/[Grafana]. For more information about building observability into your applications, see link:/docs/ref/general/#microservice_observability_metrics.html[Microservice observability with metrics].
 
-Some metric names changed between MicroProfile Metrics 1.1 and MicroProfile Metrics 2.x. Go to the link:/docs/ref/general/#metrics1.1-list.html[MicroProfile Metrics 1.1 reference list] if you need to map feature:mpMetrics-2.2[] metric information to feature:mpMetrics-1.1[] metric information.
+Some metric names changed between link:https://openliberty.io/docs/ref/feature/#mpMetrics-1.1.html[MicroProfile Metrics 1.1] and link:https://openliberty.io/docs/ref/feature/#mpMetrics.html[MicroProfile Metrics 2.x]. Go to the link:/docs/ref/general/#metrics1.1-list.html[MicroProfile Metrics 1.1 reference list] if you need to map MicroProfile Metrics 2.x metric information to MicroProfile Metrics 1.1 metric information.
+
+MicroProfile Metrics 2.3 explicitly enables the link:https://openliberty.io/docs/ref/feature/#monitor-1.0.html[Performance Monitoring 1.0 feature]. If you use earlier than version 2.3 of the MicroProfile Metrics feature, then you must also enable the Performance Monitoring 1.0 feature.
```

Comment:
"If you use earlier than version 2.3 of the MicroProfile Metrics feature, then you must also enable the Performance Monitoring 1.0 feature."
How about
"If you use a version earlier than version 2.3 of the MicroProfile Metrics feature, then you must also enable the Performance Monitoring 1.0 feature." for clarity. Just a suggestion as Acrolinx did not flag it.